src/utils/inlineAttachment.ts

```diff
@@ -9,14 +9,20 @@ import { getFilePathFromUrl } from '@/utils/signedUrlReplacer'
 
 import { getSignedUrlUpload, getSignedUrlFile } from '@/app/actions'
 
-export const uploadImageHandler = async (file: File, token: string, task_id: string | null): Promise<string | undefined> => {
+export const uploadAttachmentHandler = async (
+  file: File,
+  token: string,
+  workspaceId: string,
+  task_id: string | null,
+): Promise<string | undefined> => {
   const supabaseActions = new SupabaseActions()
-
+  const filePath = `/${workspaceId}${task_id ? `/${task_id}` : ''}`
```

Comment:
It might be better to organize scrap attachments into a separate folder like `temp/` instead of having them pile up at the root of the workspaceId folder. 

This can also help us catch edge cases.